insolar/bus/bus.go

```diff
@@ -469,7 +469,7 @@ func (b *Bus) wrapMeta(
 ) (payload.Meta, *message.Message, error) {
 	msg = msg.Copy()
 
-	meta := payload.Meta{
+	payloadMeta := payload.Meta{
 		Polymorph:  uint32(payload.TypeMeta),
 		Payload:    msg.Payload,
 		Receiver:   receiver,
```

Comment:
`Receiver` is no longer used anywhere except tests, but I did not dare to remove it from payload.Meta protobuf